Samples/SqueezeNetObjectDetection/NETCore/cs/Program.cs

```diff
@@ -117,17 +120,53 @@ private static T AsyncHelper<T> (IAsyncOperation<T> operation)
             return operation.GetResults();
         }
 
-        private static ImageFeatureValue LoadImageFile()
+        private static ImageFeatureValue LoadImageFile(ColorManagementMode colorManagementMode)
         {
             StorageFile imageFile = AsyncHelper(StorageFile.GetFileFromPathAsync(_imagePath));

```

Comment:
Can you add a try catch on loading the file with the fully qualified path error message

Reply:
Yep yep!